scripts/text_to_speech_node.py

```diff
@@ -38,10 +38,11 @@ def __init__(self, tts_module, executable=None, key=None, pitch=0):
         self.key = key
         self.pitch = pitch
 
-        self.character = rospy.get_param("~character",  "Default")
-        self.language  = rospy.get_param("~language",   "us")
-        self.voice     = rospy.get_param("~voice",      "kyle")
-        self.emotion   = rospy.get_param("~emotion",    "Neutral")
+        self.character    = rospy.get_param("~character",    "Default")
+        self.language     = rospy.get_param("~language",     "us")
+        self.voice        = rospy.get_param("~voice",        "kyle")
+        self.emotion      = rospy.get_param("~emotion",      "Neutral")
+        self.samples_path = rospy.get_param("~samples_path", os.path.expanduser("~/MEGA/media/audio/soundboard"))
```

Comment:
You should do the `os.path.expanduser` call later so people are able to use `~` in their parameters.